lineman/test/test_main.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+import lineman.__main__ as main
+from lineman.version import __version__
+
+class TestHawkEyeInterop(unittest.TestCase):
+
+    def test_proper_keys(self):
+        pre_proc = {
+            'records': {
+            'num_inputed': 0,
+            'num_redcap': 0,
+            'num_valid': 0,
+            'num_invalid': 0,
+            'validated': [],
+            'not_validated': [],
+            'mappings_done': {},
+            },
+            'subject_event_dict': {}
+        }
+        post_proc = {
+            'source': 'lineman_%s' % __version__,
```

Comment:
This is neat but could there be an issue with updating lineman and then breaking hawkeye integration? 

I suppose thats largely on hawkeye that it isnt doing string equality but is instead doing like break points or something??? This is very cool though.